lib/sheriff/plug/enforce_policy.ex

```diff
@@ -25,7 +25,15 @@ defmodule Sheriff.Plug.EnforcePolicy do
   end
 
   defp fetch_actor(conn) do
-    {conn.private[:current_user], conn}
+    case System.get_env("SHERIFF_RESOURCE_KEY") do
```

Comment:
Yeah that is better, I was brain farting on something similar to that and just got it to work with whats there now. Make the mistake too often. Will update in the morning and get everything working here in the proposal once I verify in the example app.